package.json

```
@@ -26,22 +26,23 @@
   "dependencies": {
     "@digitalbazaar/bitstring": "^1.2.0",
     "@digitalbazaar/cborld": "^4.2.0",
-    "bedrock-did-io": "^4.0.0",
     "@digitalbazaar/minimal-cipher": "^4.0.0",
     "assert-plus": "^1.0.0",
     "base58-universal": "^1.0.0",
+    "base64url-universal": "^1.1.0",
+    "bedrock-did-io": "^5.0.0",
     "bnid": "^2.0.0",
     "canonicalize": "^1.0.1",
     "cit-context": "^2.0.0",
+    "esm": "^3.2.25",
     "lru-cache": "^6.0.0",
-    "p-limit": "^3.0.1",
-    "pako": "^1.0.11"
+    "p-limit": "^3.0.1"
   },
   "peerDependencies": {
-    "bedrock": "3.x",
+    "bedrock": "^4.4.3",
     "bedrock-https-agent": "^2.0.0",
     "bedrock-mongodb": "^8.0.1",
```

Should update this as well since this will be a breaking change.

```suggestion
    "bedrock-mongodb": "^8.4.0",
```

---

Updated 5d51574